feat(localization): Add en-GB

ijxy opened the pull request:
## Description

Replaces a couple of "Americanisms" with their British versions:
- authorize with authorise
- organization with organisation


## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

tmilewski commented on .changeset/fuzzy-carrots-push.md:
```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/localizations": major
```
```suggestion
"@clerk/localizations": minor
```

ijxy replied:
Happy to continue as is, but it has occurred to me that _most_ of this file is identical to `en-US` and it would be a lot easier to maintain if only the changed values were present in this file.

As is, the `en-GB` locale would not automatically benefit from changes to the `en-US` values on the keys where they should be the same.

What do you think? Happy to strip out the identical fields if you agree.

tmilewski replied:
@ijxy That’s a good point! Yeah, let’s strip out the duplicates and we’ll merge from there.

Thanks again!

tmilewski reviewed (changes requested):
@ijxy Thank you for this! Please just update the major changeset to minor, and we should be good to go!

anagstef commented on packages/localizations/src/utils/generate.ts:
```diff
@@ -164,6 +165,7 @@ run(beBY, 'be-BY');
 run(csCZ, 'cs-CZ');
 run(daDK, 'da-DK');
 run(deDE, 'de-DE');
+run(enGB, 'en-GB');
```
Auto-generation will not work as expected if the file has custom logic inside it. 

Let's drop the changes of `generate.ts` file.

anagstef reviewed (changes requested):
Let's drop the changes of `generate.ts` file.

ijxy commented:
@anagstef @tmilewski Apologies for the delay–I have updated as per comments and extracted the changed words into a little dictionary object so it is easier to see what is actually changed.

The tl;dr is that now the enGB locale data should automatically stay up to date with any changes to enUS. 🎉

tmilewski reviewed (approved):
Hey @ijxy! We had a discussion internally and thought it'd be better to keep things consistent as well as not increase run-time work. As-such, I've applied the values in your work to the existing `en-US` file.

Thanks for your contribution!
